eng/common/TestResources/New-TestResources.ps1

```diff
@@ -223,7 +227,7 @@ function BuildBicepFile([System.IO.FileSystemInfo] $file)
     }
 
     $tmp = $env:TEMP ? $env:TEMP : [System.IO.Path]::GetTempPath()
-    $templateFilePath = Join-Path $tmp "test-resources.$(New-Guid).compiled.json"
+    $templateFilePath = Join-Path $tmp "$ResourceType-resources.$(New-Guid).compiled.json"
```

Comment:
Is there some reason we can't or shouldn't use the same? Is it just, perhaps, that perf resources are more scoped and less are likely to be needed?

Reply:
Are you asking why we can't use the same set of resources for both live tests and perf?  At least for storage, the live tests and perf tests have very different requirements.  First, live tests use one pipeline for all storage services (blob, file-share, datalake), while perf uses a separate pipeline for each service.  Second, the deployed resources are very different.  You can compare these two files:

https://github.com/Azure/azure-sdk-for-net/blob/feature/perf-automation/sdk/storage/Azure.Storage.Blobs/perf-resources.bicep

https://github.com/Azure/azure-sdk-for-net/blob/feature/perf-automation/sdk/storage/test-resources.json

While there may be some services that use the same resources for both live tests and perf tests, I think this will be rare enough it's not worth designing for.  And I think it's better to have explicit files for live and perf tests, since if someone is updating say the live test resources, they might not think to run the perf pipelines to validate.

Reply:
Fair point. Even for Key Vault we don't need - and probably don't want - a Managed HSM resources for perf tests.

Reply:
Also we ran into a problem with stress tests, where if the files are named `test-resources.json` they get auto-discovered by live tests and deployed. So we need different naming conventions for that reason.